### DISCUSSION TITLE
Is there a possibility to disable vendor prefixes?

### DISCUSSION THREAD

**Discussion by zlypher** [chakra-ui/chakra-ui#6216] [Q&A]
Chakra UI seems to add a few vendor prefixes, which might not be necessary anymore (depending on the target audience). 

![image](https://user-images.githubusercontent.com/820423/174591194-e086d7f8-7de3-4279-bbcd-3481083a4646.png)

Is there a way to disable the vendor prefixing or configure it with browserslist for example? From what I could tell, `.browserslistrc` didn't change anything. I am using Chakra UI in a NextJS project if that matters.

**Comment by JunlinPan-Crypto** [comment]
Running into same issue. Have no idea of how to disable these redundant vendor prefixes

**Comment by synasapmob** [comment]
i got the same problem, any can fix?
